frontend/src/v5/ui/routes/dashboard/projects/tickets/ticketsTable/ticketsTable.component.tsx

```diff
@@ -78,8 +78,9 @@ export const TicketsTable = () => {
 	const [sidePanelTicket, setSidePanelTicket] = useState<Partial<ITicket>>(null);
 	const [showCompleted, setShowCompleted] = useState(false);
 
-	const editingTicketId = sidePanelTicket?._id;
+	const selectedTicketId = sidePanelTicket?._id;
 	const templateIsFetched = templateAlreadyFetched(selectedTemplate || {} as any);
+	const isEditingValidTicket = sidePanelModelId && !selectedTicketId && !hasRequiredViewerProperties(selectedTemplate);
```

Comment:
I'm not sure I understand this. I take it it's supposed to be true if the user is editing a ticket which does not require the viewer for any properties. But the `!selectedTicketId` requirement would surely be true only when creating a new ticket? When testing this it doesn't seem to be the case which is confusing me.

Also is there any reason why the "Open in 3D Viewer" button should be disabled when creating a ticket that doesn't have viewer required properties?

Reply:
Sorry about the name, I am going to change that. It should have been `isCreatingValidTicket` or something similar.

As for the second question, that was the original design, that is, such a button disabled in new tickets and enabled for existing ones. I wonder if should keep it like that. @carmenfan

Reply:
> As for the second question, that was the original design, that is, such a button disabled in new tickets and enabled for existing ones. I wonder if should keep it like that. @carmenfan

It makes sense as a user flow because the whole point of that button is "go to the ticket in 3D Viewer"

If you're creating a ticket then there's nothing to go to.

Reply:
It could open the viewer with the create ticket card already open